packages/screens/Multisig/MultisigCreateScreen.tsx

```diff
@@ -43,27 +48,54 @@ import {
 } from "@/utils/style/fonts";
 import { layout } from "@/utils/style/layout";
 
-type CreateMultisigWalletFormType = {
-  addresses: { address: string }[];
-  signatureRequired: string;
-  maxSignature: string;
-  name: string;
-};
+const ZodCreateMultisigWalletFormType = z.object({
```

Comment:
```suggestion
const zodCreateMultisigWalletFormType = z.object({
```
we use capitalization only for types (in the typescript sense) and this is an object (a type schema)

Reply:
No more zod usage. Outdated